src/vehicle.h

```diff
@@ -416,6 +421,20 @@ class vehicle : public JsonSerializer, public JsonDeserializer
     vehicle();
     ~vehicle () override;
 
+    /**
+     * Set stat for part constrained by range [0,durability]
+     * @note does not invoke base @ref item::on_damage callback
+     */
+    void set_hp( vehicle_part &pt, int qty );
+
+    /**
+     * Apply damage to part constrained by range [0,durability] possibly destroying it
```

Comment:
What does "possibly destroying it" mean?
Does it mean `set_hp` above won't set some sort of a "destroyed" flag?